layouts/partials/cover.html

```diff
@@ -11,19 +11,32 @@
         {{- $processableFormats := (slice "jpg" "jpeg" "png" "tif" "bmp" "gif") }}
         {{- $prod := (hugo.IsProduction | or (eq .Site.Params.env "production")) }}
         {{- if (and (in $processableFormats $cover.MediaType.SubType) (ne .Site.Params.cover.responsiveImages false) (eq $prod true)) }}
-        <img loading="lazy" srcset="{{- range $size := $sizes -}}
-                        {{- if (ge $cover.Width $size) -}}
+        <img 
+            loading="lazy" 
+            srcset="{{- range $size := $sizes -}}
+                    {{- if (ge $cover.Width $size) -}}
```

Comment:
This is an 'unrelated change'. It's not necessary and means that no upsizing occurs, where it did in the past. I know upsizing doesn't produce ideal images, but this should at least be a change that one opts into rather than having unexpectedly occur.

Reply:
Maybe I miss the point here, but I only reformatted that part. The size check was there before my change.

Reply:
Oh, you are right, I missed that. I think the reformatting should be in a separate commit or PR, unless absolutely necessary

Reply:
Agreed. I reduced the changes to a minimum.

Reply:
Thank you. Looks good to me now. Now to see what @adityatelange thinks...

Reply:
Thanks for reviewing!